WebHostLib/api/__init__.py

```diff
@@ -56,4 +59,35 @@ def get_datapackage_checksums():
     return version_package
 
 
+@api_endpoints.route("/downloads/<string:build>")
+def get_download(build: str):
+    download_data = ArchipelagoInstaller.get(id=build)
+    if download_data is None:
+        get_latest_release()
+        download_data = ArchipelagoInstaller.get(id=build)
+    return send_file(BytesIO(download_data.data), as_attachment=True, download_name=download_data.name)
+
+
+def get_latest_release() -> None:
+    """Pulls the latest release from the Archipelago GitHub and saves them to the db."""
+    import requests
+    response = requests.get("https://api.github.com/repos/ArchipelagoMW/Archipelago/releases/latest")
+    response.raise_for_status()
+    data = response.json()
+    for asset in data["assets"]:
+        filename = asset["name"]
+        if "App" in filename:
+            name = "app"
```

Comment:
This is wrong because (once we get to it) a macos build will probably be `.app`. AppImage is AppImage, not app.